src/main/java/com/creativemd/littletiles/common/action/block/LittleActionPlaceStack.java

```diff
@@ -155,7 +155,7 @@ public PlacementResult placeTile(EntityPlayer player, ItemStack stack, World wor
                 
                 if (iTile.containsIngredients(stack)) {
                     stack.shrink(1);
-                    checkAndGive(player, inventory, getIngredients(placement.unplaceableTiles));
+                    checkAndGive(player, inventory, this.mode.getBeforePlaceIngredients(placement.unplaceableTiles));
```

Comment:
This should be left as it is, because this actually provides the left overs from the placement.